### ISSUE TITLE
JSILc added to the app menu?

### ISSUE THREAD

**Issue by cra0zy** [Protobuild/Protobuild#161]
If you wanted to make it so you can right click open with, then add 'NoDisplay=true' to .desktop file you create, that way it won't show up in the app menu.

Also add an icon file: http://www.luminance.org/images/jsil.png

Also also it would be much more useful if you added it as a terminal command instead of launcher...

**Comment by hach-que** [comment]
Yeah, the difficulty is that we currently don't designate tools as either graphical tools or command-line tools, so all global tools get added to the start menu / application menu / etc. at the moment.

**Comment by hach-que** [comment]
To resolve this we need to support an additional property for projects that are installed - an `IsGraphicalTool` property.  Only projects that are installed globally with this flag will get added to the menu.

**Comment by hach-que** [comment]
We intend on replacing JSIL with an alternative (like Bridge.NET), so I'm closing this issue as Won't Fix.

**Comment by cra0zy** [comment]
Technically this request has nothing to do with JSIL:
> Yeah, the difficulty is that we currently don't designate tools as either graphical tools or command-line tools, so all global tools get added to the start menu / application menu / etc. at the moment.

**Comment by hach-que** [comment]
Good point. I'll retitle it.
